commercial/app/views/contentapi/itemsV2.scala.html

```diff
@@ -3,14 +3,53 @@
   optCapiTitle: Option[String],
   optCapiLink: Option[String],
   optCapiAbout: Option[String],
-  optClickMacro: Option[String],
-  optOmnitureId: Option[String],
+  clickMacro: Option[String],
+  omnitureId: Option[String],
   optCapiAdFeature: Option[String],
   optSponsorType: Option[String],
   optSponsorLabel: Option[String])(implicit request: RequestHeader)
 
-@containerWrapper(Seq(), "")(HtmlFormat.empty)(HtmlFormat.empty)(HtmlFormat.empty)(HtmlFormat.empty) {
+@import views.html.fragments.inlineSvg
 
+@defining(("1_0", "2014-08-14")) { case (version, date) =>
+    @containerWrapper( Seq("legacy", "capi", "tone-capi")
+                     , optBadge = Some(badge)
+                     , optKicker = Some(fragments.commercial.paidForMeta())
+                     , optStamp = Some(stamp)
+                     ){
 
+        @optCapiLink.map { linkUrl =>
+            <a href="@clickMacro@linkUrl" class="adverts__logo u-text-hyphenate" data-link-name="header link">@optCapiTitle</a>
+        }.getOrElse {
+            <span class="u-text-hyphenate">@optCapiTitle</span>
+        }
+
+    }{
+
+        @items.map { item => @itemCard(item, version, date, clickMacro) }
+
+    }
+}
+
+@stamp = {
+
+    <a @if(Edition(request).id == "AU") {
+            href="@LinkTo("/guardian-labs-australia")"
+        } else {
+            href="@LinkTo("/guardian-labs")"
+        }
+        >
+        @inlineSvg("glabs-logo", "logo")
+        <span class='u-h'>Guardian Labs</span>
+    </a>
+
+}
+
+@badge = {
+
+    @optSponsorLabel
+    <a href="@clickMacro@optCapiLink" data-link-name="logo link">
+        @for(logoUrl <- optLogo) {<img class="adverts__badge__logo" src="@logoUrl" alt="">}
```

Comment:
Element of element? Can we figure out something else? Should we possibly treat badge as a separate module? We have badges all over the page so maybe it can be done.